src/main/cpp/util/InstallDirs.cpp

```diff
@@ -167,5 +170,21 @@ path InstallDirs::GetRootDir()
 	return g_root_dir;
 }
 
+/// Reads the XML file at the given path, relative to some other path.
```

Comment:
Actually, I suppose we should remove this bit of documentation and only keep the copy in the header file.